pkg/catalog/manager/crd.go

```diff
@@ -172,7 +168,7 @@ func (m *Manager) createTemplateVersions(catalogName string, versionsSpec []v3.T
 		templateVersion := &v3.CatalogTemplateVersion{}
 		templateVersion.Spec = spec
 		templateVersion.Status = v3.TemplateVersionStatus{HelmVersion: template.Status.HelmVersion}
-		templateVersion.Name = fmt.Sprintf("%s-%v", template.Name, spec.Version)
+		templateVersion.Name = getValidHelmChartLabel(template.Name, spec.Version)
```

Comment:
Should this be passing through the label validation?

Reply:
I added label validation to the template name and remove it from the template version name